v63002/u_inref/gtm8694.csh

```diff
@@ -126,15 +126,21 @@ TRIGGER_MOD:$groupname
 EOF
 chmod -w $ydb_dist/restrict.txt
 $gtm_tst/com/lsminusl.csh $ydb_dist/restrict.txt | $tst_awk '{print $1,$9}'
-setenv EDITOR ""
+set sedpath=`which sed`
+setenv EDITOR $sedpath
 $ydb_dist/mumps -run breakfn^gtm8694 >& unrestrictedBREAK.txt
 cat unrestrictedBREAK.txt
 $ydb_dist/mumps -run zbreakfn^gtm8694 >& unrestrictedZBREAK.txt
 cat unrestrictedZBREAK.txt
 $ydb_dist/mumps -run zcmdlnefn^gtm8694 "ZCMDLNE was not ignored" >& unrestrictedZCMDLINE.txt
 cat unrestrictedZCMDLINE.txt
+echo "# TESTING ZEDIT"
 $ydb_dist/mumps -run zeditfn^gtm8694 >& unrestrictedZEDIT.txt
-cat unrestrictedZEDIT.txt
+if ("" == `$grep RESTRICTEDOP unrestrictedZEDIT.txt`) then
```

Comment:
As discussed, please include a comment as to WHY this change was needed. Either here in the script or in the commit message detail.